parquet2hive_modules/parquet2hivelib.py

```diff
@@ -102,18 +105,26 @@ def get_bash_cmd(location, success_only=False, recent_versions=None, version=Non
 
         partitions = get_partitioning_fields(key.key[len(prefix):])
 
-
-        default_table_name = _normalize_table_name(dataset_name)
         version_table_name = _normalize_table_name(dataset_name + "_" + version)
-        bash_cmd += "hive -e '{}'\n".format(parquet2sql(schema, version_table_name, version_location, partitions))
+        version_sql = parquet2sql(schema, version_table_name, version_location, partitions)
+        if just_sql:
+            output += version_sql + "\n"
+        else:
+            output += "hive -e {}\n".format(shlex_quote(version_sql))
+
         if versions_loaded == 0:  # Most recent version
-            bash_cmd += "hive -e '{}'\n".format(parquet2sql(schema, default_table_name, version_location, partitions))
+            default_table_name = _normalize_table_name(dataset_name)
+            default_sql = parquet2sql(schema, default_table_name, version_location, partitions)
+            if just_sql:
+                output += default_sql
+            else:
+                output += "hive -e {}\n".format(shlex_quote(default_sql))
```

Comment:
Why don't you pull this to a separate function? We should have done that to begin with.